esx_service/vmdk_ops.py

```diff
@@ -427,9 +442,15 @@ def removeVMDK(vmdk_path):
         elif rc != 0:
             return err("Failed to remove %s. %s" % (vmdk_path, out))
         else:
+            # remove succeed, remove infomation of this volume from volumes table
+            if tenant_uuid:
+                error_info = auth.remove_volume_from_volumes_table(tenant_uuid, datastore, vol_name)
+                return error_info
+            else:
+                if not vm_name:
+                    logging.warning("VM %s does not belong to any tenant", vm_name)
```

Comment:
In this PR, I just define a global for format string. But in the long run, I think we need to have a file which defines all the error code and corresponding error string for all the error message we used in our code.